### DISCUSSION TITLE
Welcome to runitor Discussions!

### DISCUSSION THREAD

**Discussion by bdd** [bdd/runitor#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SkYNewZ** [comment]
Hi! This is what I build with your service: https://github.com/SkYNewZ/docker-runitor. It's scheduled so always on the latest version. Uploaded on [Docker Hub](https://hub.docker.com/repository/docker/skynewz/runitor/tags).

I built it to use with another project: https://github.com/SkYNewZ/docker-feedly-restic-backup.

So, thanks for you [healthchecks.io](http://healthchecks.io/) helper ! 😄
